fastapi_code_generator/parser.py

```diff
@@ -274,7 +280,10 @@ def get_parameter_type(
         if not data_type:
             if not schema:
                 schema = parameters.schema_
+            if schema is None:
+                raise RuntimeError("schema is None")  # pragma: no cover
             data_type = self.parse_schema(name, schema, [*path, name])
+            data_type = self._collapse_root_model(data_type)
```

Comment:
Obviously, if it were datamodel-code-generator, I would make it optional, but for this project, it should be standard for convenience reasons.